templates/blog/index.html

```diff
@@ -2,7 +2,7 @@
 {% set active_page = "blog" %}
 {% block outer_content %}
 
-<section id="main-content" class="p-strip--image is-shallow">
+<section class="p-strip is-shallow">
```

Comment:
The id "main-content" is linked to by accessibility aids (search for it in the page source) so cannot be removed.

Reply:
oh, "duplicate" I misread... all ok!